resqpy/rq_import/__init__.py

```diff
@@ -0,0 +1,14 @@
+"""Functions for RESQML objects related to importing"""
+
+all = [
+    '_import_nexus.py', '_GridFromCp.py', '_import_vdb_all_grids.py', '_grid_from_cp.py', '_import_vdb_ensemble.py',
+    '_add_ab_properties.py', '_add_surfaces.py'
+]
+
+from ._GridFromCp import GridFromCp
```

Comment:
Please remove this import here (will necessitate some changes elsewhere to resqpy code importing GridFromCp).